bueno logo

Lcjacobpd opened the pull request:
updated local fork, added bueno logos at docs/img/ and included it in main readme

Lcjacobpd committed:
build & run readme

+ added README.md to build-and-run example

Commit:
Squashed commit of the following:

commit 26ca229bd0e50a2cb6d23eecd9d6a2eda984f9e1
Author: Jacob Dickens <lcjacobpd@localhost.localdomain>
Date:   Fri Oct 2 09:31:35 2020 -0600

    Resized Logos

    = resized bueno logos in docs/img/

commit ba43937ffc23e516aea8c3d79a802f0dcecc3a3f
Author: Jacob Dickens <lcjacobpd@localhost.localdomain>
Date:   Fri Oct 2 09:13:51 2020 -0600

    Bueno Logo

    + added 3 logos to docs/img/
    + added logo think to main readme

Commit:
Squashed commit of the following:

commit 26ca229bd0e50a2cb6d23eecd9d6a2eda984f9e1
Author: Jacob Dickens <lcjacobpd@localhost.localdomain>
Date:   Fri Oct 2 09:31:35 2020 -0600

    Resized Logos

    = resized bueno logos in docs/img/

commit ba43937ffc23e516aea8c3d79a802f0dcecc3a3f
Author: Jacob Dickens <lcjacobpd@localhost.localdomain>
Date:   Fri Oct 2 09:13:51 2020 -0600

    Bueno Logo

    + added 3 logos to docs/img/
    + added logo think to main readme

Commit:
Merge with Upstream

Commit:
Merge branch 'master' of github.com:Lcjacobpd/bueno

samuelkgutierrez commented on examples/build-and-run/README.md:
````diff
@@ -8,12 +8,20 @@ for nbody, our example application.
 bueno build -s Dockerfile.mpich -t nbody
 ```
 Once completed, there will be a new tarball in the acting directory. We'll use
+<<<<<<< HEAD
````
It looks like you didn't resolve merge conflicts. This is an example of one of them.

samuelkgutierrez reviewed (changes requested):
* Please include the vector graphics image for the logo
* Resolve conflicts in examples/build-and-run/README.md

Lcjacobpd commented:
I'll double check, I thought I'd resolved that already

On Fri, Oct 2, 2020, 4:52 PM Samuel K. Gutiérrez <notifications@github.com>
wrote:

> *@samuelkgutierrez* commented on this pull request.
> ------------------------------
>
> In examples/build-and-run/README.md
> <https://github.com/lanl/bueno/pull/9#discussion_r499080482>:
>
> > @@ -8,12 +8,20 @@ for nbody, our example application.
>  bueno build -s Dockerfile.mpich -t nbody
>  ```
>  Once completed, there will be a new tarball in the acting directory. We'll use
> +<<<<<<< HEAD
>
> It looks like you didn't resolve merge conflicts. This is an example of
> one of them.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lanl/bueno/pull/9#pullrequestreview-501485474>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJTH6ZYSCNUUI6LUSBW2ODLSIZKSHANCNFSM4SB4M6BA>
> .
>

Commit:
Resolve Merge and SVG

+ resolved merge conflict in examples/build-and-run/README.md
+ added svg file for icon in docs/img

samuelkgutierrez commented on examples/build-and-run/README.md:
```diff
@@ -8,20 +8,12 @@ for nbody, our example application.
-=======
-Instead of simply ordering bueno to execute our ruscript as we have in the
->>>>>>> c26a6a224e02ebd1914befe20014fa2d2a828bca
+Instead of simply ordering bueno to execute our runscript as we have in the
```
This wasn't the version at HEAD.

samuelkgutierrez reviewed (changes requested):
Thank you, @Lcjacobpd 

Almost got it, but you merged the incorrect version in one spot (I'll mark it).

Commit:
build-and-run/README.md

+ pulled markdown from upstream (discard local)

samuelkgutierrez reviewed (approved):
Looks good. Thank you @Lcjacobpd
